### PR TITLE
etcd/live-reconfiguration: Clean up style, language atop #749

### PR DESCRIPTION
**Pull request by joshix**
Cleanup edits/rewrite to the entire live-reconf document.
Incremental; second person not entirely removed and can still
stand to be clarified further (with procedure testing) in some
sections.

### CODE REVIEW

File: etcd/etcd-live-cluster-reconfiguration.md [hunk 1]
```diff
@@ -101,40 +101,34 @@ member 60e8a32b09dc91f1 is healthy: got healthy result from http://10.0.1.3:2379
...
+Check that the `/var/lib/etcd2/` directory exists and is empty. If you removed this directory accidentally, you can recreate it with the proper modes by using:
 
 ```sh
 $ sudo systemd-tmpfiles --create /usr/lib64/tmpfiles.d/etcd2.conf
 ```
 
-or with this one:
-
-```sh
-$ sudo install -m 755 -d -o etcd -g etcd /var/lib/etcd2
```
Comment (kayrus) [inline etcd/etcd-live-cluster-reconfiguration.md]: Why did you remove this command? What if etcd2.conf name will be changed?

File: etcd/etcd-live-cluster-reconfiguration.md [hunk 2]
```diff
@@ -204,22 +196,22 @@ $ sudo systemctl stop etcd2
 
 If you have etcd proxy nodes, they should update members list automatically according to the [`--proxy-refresh-interval`][proxy-refresh] configuration option.
 
-Then on one of the *member* node we have to run following command to backup current [data directory][data-dir]:
+Then, on one of the *member* nodes, run the following command to backup the current [data directory][data-dir]:
 
 ```sh
 $ sudo etcdctl backup --data-dir /var/lib/etcd2 --backup-dir /var/lib/etcd2_backup
```
Comment (kayrus) [inline etcd/etcd-live-cluster-reconfiguration.md]: I suggested to create a backup here

File: etcd/etcd-live-cluster-reconfiguration.md [hunk 3]
```diff
@@ -229,25 +221,27 @@ member e6c2bda2aa1f2dcf is healthy: got healthy result from http://10.0.1.2:2379
...
-If output messages don't contain any error you have to remove `/run/systemd/system/etcd2.service.d/98-force-new-cluster.conf` drop-ip and reload systemd daemon: `sudo systemctl daemon-reload` (`etcd2` daemon restart is not necessary).
+If the output contains no errors, remove the `/run/systemd/system/etcd2.service.d/98-force-new-cluster.conf` drop-in file, and reload systemd services: `sudo systemctl daemon-reload`. It is not necessary to restart the `etcd2` service after this step.
+
+The next steps are those described in the [Change the etcd Cluster Size][change-cluster-size] section, with one difference: Remove the `/var/lib/etcd2/member` directory as well as `/var/lib/etcd2/proxy`.
 
-Next steps are similar to the steps described in the [Change the etcd Cluster Size][change-cluster-size] section but with one exception: you have to remove `/var/lib/etcd2/member` directory as well as `/var/lib/etcd2/proxy` one. **NOTE**: It is strongly recommended to make a backup before you remove these directories:
+It is strongly recommended to make a backup before you remove these two directories:
 
 ```sh
 $ sudo etcdctl backup --data-dir /var/lib/etcd2 --backup-dir /var/lib/etcd2_backup
```
Comment (kayrus) [inline etcd/etcd-live-cluster-reconfiguration.md]: And here. Should we remove one note?

File: etcd/etcd-live-cluster-reconfiguration.md (hunk 1 above)
Comment (joshix) [inline etcd/etcd-live-cluster-reconfiguration.md]: The goal of the edit was to present one way to do it, rather than two. But you make a fair point -- the invocation of `install` shows the desired mode on the directory. Of course, so would, e.g., `mkdir -p /var/lib/etcd2 && chown -R etcd:etcd !$ && chmod 0755 !$`

So I would like to choose one "official" way -- and I think the systemd utility approach is the most consistent with our "way" of doing service management.

Now that I have explained my thinking, do you think we still need both?

File: etcd/etcd-live-cluster-reconfiguration.md (hunk 3 above)
Comment (joshix) [inline etcd/etcd-live-cluster-reconfiguration.md]: Yep, good catch. I think we remove the second one, because if you've made it through creating a one-node cluster, you don't really need to do the backup yet again.

File: etcd/etcd-live-cluster-reconfiguration.md (hunk 2 above)
Comment (joshix) [inline etcd/etcd-live-cluster-reconfiguration.md]: (resolved with edit below)

File: etcd/etcd-live-cluster-reconfiguration.md (hunk 1 above)
Comment (joshix) [inline etcd/etcd-live-cluster-reconfiguration.md]: @kayrus I'm going to merge without adding the second way back; If you think it really needs to be there, let me know and we'll add the `install` invocation back. But I prefer to just update the doc should the etcd2.conf file change in future, and have just "one way to do it" in the doc.
